content/backend/graphql-python/8-pagination.md

```diff
@@ -31,17 +31,17 @@ class Query(graphene.ObjectType):
 
     # Use them to slice the Django queryset
     def resolve_links(self, info, search=None, first=None, skip=None, **kwargs):
-        qs = Link.objects.all()
-
         if search:
             filter = (
                 Q(url__icontains=search) | 
                 Q(description__icontains=search)
             )
             qs = qs.filter(filter)
+        else:
```

Comment:
I think the code is cleaner without the `else` statement. The point is that, we are always going to get all objects, and apply filters as needed.